native/cocos/audio/android/audio_utils/include/audio_utils/minifloat.h

```diff
@@ -18,7 +18,11 @@
 #define COCOS_AUDIO_MINIFLOAT_H
 
 #include <stdint.h>
+
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
 #include <sys/cdefs.h>
+#endif
+#include "audio/android/utils/Compat.h"
```

Comment:
Why do we need Compat here?

Reply:
Openharmony's __BEGIN_DECLS definition is here